corehq/toggles/__init__.py

```diff
@@ -690,6 +690,13 @@ def _ensure_valid_randomness(randomness):
               'spaceKey=saas&title=Allow+Configuration+of+Case+List+Tiles',
 )
 
+DATASOURCE_BACKGROUND_REFRESH = StaticToggle(
```

Comment:
@kaapstorm Putting this behind a FF. Feel free to suggest otherwise.

Reply:
I think the decision was to use the existing `SUPERSET_ANALYTICS` FF.